vonk/overview.rst

```diff
@@ -0,0 +1,93 @@
+.. _vonk_overview:
+
+Overview of Vonk FHIR Server, Plugins and Facades
+=================================================
+
+Framework
+---------
+
+Vonk is not just a FHIR Server, it is a processing pipeline for handling standard and custom FHIR requests. :ref:`Vonk FHIR Server <vonk_overview_server>` consists of this pipeline filled with processors to handle the interactions defined in the FHIR RESTful API. With :ref:`vonk_overview_plugins` you can add your own processors to the framework to perform custom operations, or fill in cross-cutting concerns for your business. A :ref:`Facade <vonk_overview_facades>` is a type of plugin that provides a data access layer for an existing data repository (e.g. a proprietary system). This image sums it all up:
+
+.. image:: ./images/vonk_framework.png
+  :align: center
+
+Vonk comes in several editions:
+
+* Sandbox: try it right now on https://vonk.fire.ly
+* Evaluation: get an evaluation license from `Simplifier.net <https://simplifier.net/vonk>`_, allowing you to explore all functionality for free during a month (renewable)
+* Community: use Vonk for free, but only on SQLite
+* Commercial use with professional support in different scales: Startup, Growth, Scale and Enterprise 
+
+.. TODO: license link to Simplifier for Community 
+
+For more information and pricing visit the `product site <https://fire.ly/products/vonk/>`_.
+
+.. _vonk_overview_server:
+
+Complete Server
+---------------
+
+Vonk FHIR Server is a FHIR Server out of the box. It is built in Microsoft .NET Core and runs on any of the platforms for which a `.NET Core Runtime <https://dotnet.microsoft.com/download>`_ is available. Linux, Windows, MacOS, Docker etcetera. Installation can be done in minutes. After that you can configure main features and further details:
+
+* Choose your database: SQLite is configured by default, but for serious use you'd want to configure MongoDB or SQL Server.
+* Configure the level of validation: Vonk can be very loose or very strict on the resources that you send to it.
+* Configure endpoints for FHIR versions that you want to support (since Vonk 3.0.0, FHIR STU3 and FHIR R4)
+* Fill in your licensefile.
+* Adjust the processing pipeline by trimming it down (excluding certain plugins) or extending it with extra plugins.
+
+Besides configuration of the settings, Vonk features an Administration API that allows you to configure the so-called Conformance Resources that drive parsing, serialization, validation and terminology. The Administration API is pre-filled with all the StructureDefinitions, Searchparameters, CodeSystems and ValueSets that come with the FHIR Specification. Beyond that you can use the Administration API to make Vonk aware of:
```

Comment:
```suggestion
Besides configuration of the settings, Vonk features an Administration API that allows you to configure the so-called Conformance Resources that drive parsing, serialization, validation and terminology. The Administration API is pre-filled with Conformance Resources such as the StructureDefinitions, Searchparameters, CodeSystems and ValueSets that come with the FHIR Specification. Beyond that you can use the Administration API to make Vonk aware of:
```

Just to make it clearer to our FHIR newbies - tie definition of conformance resources to actual resource names